submission/programs/adae.R

```diff
@@ -0,0 +1,350 @@
+# Name: ADAE
+#
+# Label: Adverse Event Analysis Dataset
+#
+# Input: ae, adsl
+# Output: adae.xpt
+#
+# Developed by : [Author/Developer name]
+
+library(admiral)
+library(admiral.test) # Contains example datasets from the CDISC pilot project
+library(curl)
+library(dplyr)
+library(tidyr)
+library(lubridate)
+library(stringr)
+library(arsenal)  
+library(diffdf)
+library(xportr)
+library(metacore)
+library(metatools)
+
+rm(list = ls())  # Code removes all the Objects in the envirnoment 
+
+# ----------------------------------------------------------------------------#
+# Use e.g. haven::read_sas to read in .sas7bdat, or other suitable functions  #
+# as needed and assign to the variables below.                                #
+# For illustration purposes read in admiral test data                         #
+# ----------------------------------------------------------------------------#
+
+# ---------- #
+# read in AE #
+# ---------- #
+ae <- haven::read_xpt("https://github.com/RConsortium/submissions-pilot3-adam/blob/main/sdtm/ae.xpt?raw=true")
+suppae <- haven::read_xpt("https://github.com/RConsortium/submissions-pilot3-adam/blob/main/sdtm/suppae.xpt?raw=true")
+
+# ------------ #
+# read in ADSL #
+# ------------ #
+adsl <- haven::read_xpt("https://github.com/RConsortium/submissions-pilot3-adam/blob/main/submission/datasets/adsl.xpt?raw=true")
```

Comment:
Please don't download the files but instead read from the `sdtm` folder.

Reply:
updated to 
```
# ---------- #
# read in AE #
# ---------- #
ae <- haven::read_xpt(file.path("sdtm", "ae.xpt"))
suppae <- haven::read_xpt(file.path("sdtm", "suppae.xpt"))

# ------------ #
# read in ADSL #
# ------------ #
adsl <- haven::read_xpt(file.path("submission", "datasets", "adsl.xpt"))
```
Please review and resolve if update looks good.

Reply:
@SHAESEN2 or @thomas-neitmann If ok, please `Resolve conversation`.